apps/web/src/pages/api/payment/stripe/[publickey].ts

```diff
@@ -0,0 +1,77 @@
+import { NextApiRequest, NextApiResponse } from "next"
+import { getStripeObject, getStripeWebhookSecret } from "@dotkomonline/api/src/lib/stripe"
```

Comment:
should probably export the whole handler in @dotkomonline/api and export it. Then you should just import the handler and do

```js
import { paymentHandler } from "@dotkomonline/api"

export default paymentHandler
```

Reply:
Fixed